blobfile/_ops.py

```diff
@@ -62,6 +61,7 @@ def configure(
     get_http_pool: Optional[Callable[[], urllib3.PoolManager]] = None,
     use_streaming_read: bool = False,
     default_buffer_size: int = DEFAULT_BUFFER_SIZE,
+    save_access_token_to_disk: bool = True,

```

Comment:
I think this my my final request here, but this should default to `False` in case there is some issue, it can be changed to `True` later if nobody has any issues with it.

Reply:
Sorry, didn't notice this earlier.